contracts/governance/foundry.toml

```diff
@@ -0,0 +1,7 @@
+[profile.default]
+src = 'src'
+out = 'out'
+libs = ['lib']
+fs_permissions = [{ access = "read", path = "./"}]
+remappings = ['@openzeppelin/=./lib/openzeppelin-contracts/contracts/']
+optimizer_runs = 20_000
```

Comment:
The default is 200 but since we're optimising for function runs rather than deployment size I've pushed up to 20K. Ran tests to see the difference between 20K and 1m and there's really no gas difference but deployment size is affected a little so I'd like to leave at 20K.

Reply:
The new IR optimiser made a difference though so turned it on [here](https://github.com/0xProject/protocol/pull/641/commits/586f509f20d9658f2543057fe31ca2be7ca7c8ef)

Reply:
I've had to revert this as unit test `testShouldNotBeAbleToDelegateWithSignatureAfterExpiry` starts failing and needs to be investigated further.

Reply:
I could put in alternative compilation settings for unit tests but testing it with a different compiler than production worries me so I'd rather not switch that on until the issue is fixed https://github.com/foundry-rs/foundry/issues/1373

Reply:
The specific usage we have in `testShouldNotBeAbleToDelegateWithSignatureAfterExpiry` can be fixed by setting the expiry to `block.timestamp - 1` and avoiding the use of `vm.warp` altogether. You can see what I've done in #674

Reply:
Cherry-picked your commit https://github.com/0xProject/protocol/pull/674/commits/87b0e8b90789beeb3c3c8eabe4e282c2dd4ac464

Reply:
Just leaving a note here as I found another strange side effect with timestamp demonstrated in the following code:

```
        uint256 lastTimestamp = block.timestamp; // e.g. if block.timestamp = 1 => lastTimestamp = 1

        vm.warp(lastTimestamp + 100); // sets block.timestamp = 101 but also sets lastTimestamp = 101 !?
```

Anyway I have gone around it but worth noting as it goes against the rules of variable assignement.